doc/release_notes.rst

```diff
@@ -32,39 +23,1096 @@ This version 0.1 release has the following software components:
+
+  - GVT-g for ACRN (a.k.a ACRNGT): GPU sharing feature enabled. Service OS
+    and User OS can run GPU workload simultaneously.
+  - Direct display supported by ACRNGT. Service OS and User OS are each
+    assigned to different display. The display ports supports eDP, HDMI.
+  - Surface sharing supported by ACRNGT. SOS can access an individual
+    surface or a set of surfaces from UOS without having to access the
+    entire frame buffer of UOS.
+  - See :ref:`APL_GVT-G-hld` documentation for more information.
+
+* Virtio network supported. SOS and UOS can share physical LAN network
+  and physical eMMC storage device.  (See :ref:`virtio-hld` for more
+  information.)
+* Device pass-through to UOS support for:
+
+  - USB controller
+  - SD card
+  - IPU
+  - Bluetooth
+  - Audio
+  - Converged Security Engine (CSE)
+
+* Developers can configure hypervisor via Kconfig parameters. (See
+  :ref:`configuration` for configuration options.)
+* New ACRN tools support, including acrnctl, acrntrace, acrnlog,
+  acrn-crashlog, acrnprobe. (See :ref:`tools` documentation for details.)
+* Managed the Service OS lifecycle
+* virtual Slim Boot Loader (vSBL) for Android support
 
 Known Issues
 ============
 
-* GPU - Preemption (Prioritized Rendering,
-* Batch Preemption, QoS Rendering)
-* Preemption feature works, but performance is not optimized yet.
-* Wifi not supported in guest OS
-* Audio pass-through to guest OS, but can only be validated on driver level
-  using command line or alsa-player, and only supports limited formats
-* CSME pass-through to guest OS
-* GVT-g is available, need to perform features after configured properly
-* Camera works in Android guest. But camera app may crash after 5 mins
-* SD card works in Android guest, but does not support hot-plug.
-* Surface Sharing: Sometimes the window setup on Service OS takes up to 30 second
-* Sometimes system hangs, especially when workload is high (e.g. running benchmarks, playing videos)
+Black screen displayed after booting SOS/UOS
+  The ``weston`` display server, window manager, and compositor used by ACRN
+  (from Clear Linux) may not have been properly installed and started.
+  **Workaround** is described in ACRN GitHub issue :acrn-issue:`663`.
+
+SSD Disk ID not consistent between SOS/UOS
+  The SSD disk ID in the UOS is not the same as in the SOS when
+  the SSD device is pass-through to the UOS. The ID is also changing after
+  a reboot.  This shouldn't' be the case.  **Workaround:** None.
+  There is no impact to functionality. The issue will be fixed in the
+  next release.
+
+Hypervisor and DM version numbers incorrect
+  The command ``/usr/bin/acrn-dm --version`` does not show the correct
+  build number.  **Workaround":** None.  The issue will be fixed in the
+  next release.
+
+
+Change Log
+==========
+
+These commits have been added to the acrn-hypervisor repo since our
+initial release in March 2018 (click on the CommitID link to see details):
```

Comment:
You say "March 2018" here but the comment below indicates "--after="2018-05-11""?

Reply:
That's because there's no git history before Fri May 11 14:44:28 2018.  I'll just delete mentioning March 2018 here.